cross/minio/Makefile

```diff
@@ -1,5 +1,6 @@
 PKG_NAME = minio
-PKG_VERS = 2020-11-25T22-36-25Z
+PKG_VERS = 2021-06-14T01-29-23Z
+PKG_VERS_TIMESTAMP = 2021-06-14T01:29:23Z
```

Comment:
I have just a proposal to avoid copy-paste mistake: generate PKG_VERS from PKG_VERS_TIMESTAMP with ':' character replacement to '-'

Reply:
I have already done this but not yet pushed, as I wanted to redesign shared folders too.
So I will do an intermedeate commit.